app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    result = split_integer(32, 6)
+    assert sum(result) == 32
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    result = split_integer(6, 2)
+    assert result == [3, 3]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    result = split_integer(100, 1)
+    assert result == [100]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    result = split_integer(32, 6)
+    assert result == [5, 5, 5, 5, 6, 6]
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    result = split_integer(5, 8)
+    assert result == [0, 0, 0, 1, 1, 1, 1, 1]
```

Comment:
in this case u don't need to create a separated variable for 1-time use
instead of
```
result = split_integer(6, 2)
assert result == [3, 3]
```
do
```
assert split_integer(6, 2) == [3, 3]
```
etc.

Reply:
Done